src/main/daml/Daml/Finance/Interface/Instrument/Swap/Fpml/Factory.daml

```diff
@@ -0,0 +1,59 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Interface.Instrument.Swap.Fpml.Factory where
+
+import Daml.Finance.Interface.Instrument.Base.Instrument qualified as Instrument (I)
+import Daml.Finance.Interface.Instrument.Swap.Fpml.Types (Fpml)
+import Daml.Finance.Interface.Types.Common (InstrumentKey, PartiesMap)
+import Daml.Finance.Interface.Util.Disclosure qualified as Disclosure (I, Implementation)
+
+-- | Type synonym for `Factory`.
+type F = Factory
+
+-- | Type synonym for `View`.
+type V = View
+
+-- | View of `Factory`.
+data View = View
+  with
+    provider : Party
+      -- ^ The provider of the `Factory`.
+  deriving (Eq, Ord, Show)
```

Comment:
Do we need `Ord`? There is an item to remove unnecessary `Ord` instances (https://github.com/digital-asset/daml-finance/issues/208), so just checking if this one is required or not

Reply:
It seems to work without it, at least Daml Studio does not show an error. However, these instrument views all have `Ord` at the moment. If desired, probably better if somebody removes them all at once in a controlled way rather than having most views with `Ord` and a couple without.

Reply:
Not a biggie, so https://github.com/digital-asset/daml-finance/pull/224

Reply:
Thanks!